Angular/src/app/dialogs/document-filter/document-filter.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { MatDialogRef } from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-document-filter',
+  templateUrl: './document-filter.component.html',
+  styleUrls: ['./document-filter.component.scss'],
+})
+export class DocumentFilterComponent {
+  form: any = new FormGroup({
+    author: new FormControl(''),
+    title: new FormControl(''),
+    editor: new FormControl(''),
+    status: new FormControl(''),
+  });
+
+  constructor(public dialogRef: MatDialogRef<DocumentFilterComponent>) {}
+
+  public test() {
+    console.log(this.form.value);
+  }
+
+  protected close_dialog() {
```

Comment:
Ik zou deze eerder iets zoals submit_filter noemen, want het is nu ook mogelijk om de dialog te closen zonder deze functie aan te roepen nietwaar?

Reply:
Ik was idd niet tevreden over deze naam! `submit_filter` is veel beter!